test/integration/object_storage_buckets_test.go

```diff
@@ -93,6 +93,65 @@ func TestListObjectStorageBuckets(t *testing.T) {
 	}
 }
 
+func TestGetObjectStorageBucketAccess(t *testing.T) {
+	corsEnabled := false
+
+	// Inject ACL and Cors config into create opts
+	oldCreateOpts := testObjectStorageBucketCreateOpts
+	defer func() { testObjectStorageBucketCreateOpts = oldCreateOpts }()
```

Comment:
I would just construct a new `createOpts` here instead, it would be cleaner. Also, each test runs in a separate goroutine so all tests that reference the `testObjectStorageBucketCreateOpts` could have your modified options.

Reply:
Yeah, that felt kind of hacky. I guess we might need to adapt `setupObjectStorageBucket` unless there's a better way to do it that I'm missing.